src/main/java/Event.java

```diff
@@ -0,0 +1,33 @@
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class Event extends Task {
+
```

Comment:
It would be good if you can add JavaDoc for src/main/java/Event.java, src/main/java/DukeNoArgumentsException.java and other files. But other than that your code is very clear and concise. ^ ^